### PR TITLE
feat: pr/issue templates

### PR DESCRIPTION
**Pull request by ozwaldorf**
# Why

Currently there are no templates provided for issues and PRs

# What

- add PR template
- add issue template

## Demo

image

# Notes

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Environment
+
+Please specify any relevant setup details. For example host OS, natively run node or docker, etc.
```
Comment (kckeiks) [inline .github/issue_template.md:3]: Can we have these as comments?

Comment (ozwaldorf) [reply .github/issue_template.md:3]: > do you mean quotes? 

like that?

**Comment by kckeiks** [comment]
Will a small checklist be helpful? Something like

* I have tested this change.
* I have added unit tests.

File: .github/issue_template.md (hunk 1 above)
Comment (kckeiks) [reply .github/issue_template.md:3]:
```
<!---
Description
-->
```

Currently, users will have to manually delete these so they don't show up in PR. Including them as comments means users can see them when opening a PR but they don't have to manually delete them.  It's just makes it easier.

Comment (ozwaldorf) [reply .github/issue_template.md]: Ended up using html comments: https://github.com/fleek-network/ursa/pull/265/commits/4b57fb05dd1540da14921b36ca67a13a72bccdff

**Review by kckeiks — APPROVED** [review]

**Pull request merged**